03-setup-ingress-controller/ambassador.md

```diff
@@ -80,34 +80,38 @@ Steps to follow:
     ```shell
     helm repo add datawire https://www.getambassador.io
 
+    helm repo update datawire
+
     helm search repo datawire
     ```
 
     The output looks similar to the following:
 
     ```text
-    NAME                            CHART VERSION   APP VERSION     DESCRIPTION                                       
-    datawire/ambassador             6.7.13          1.13.10         A Helm chart for Datawire Ambassador              
-    datawire/ambassador-operator    0.3.0           v1.3.0          A Helm chart for Kubernetes                       
-    datawire/telepresence           2.4.0           2.4.0           A chart for deploying the server-side component...
+    NAME                         CHART VERSION  APP VERSION  DESCRIPTION
+    datawire/ambassador          6.9.3          1.14.2       A Helm chart for Datawire Ambassador
+    datawire/ambassador-operator 0.3.0          v1.3.0       A Helm chart for Kubernetes
+    datawire/edge-stack          7.2.0          2.1.0        A Helm chart for Ambassador Edge Stack
+    datawire/emissary-ingress    7.2.0          2.1.0        A Helm chart for Emissary Ingress
+    datawire/telepresence        2.4.9          2.4.9        A chart for deploying the server-side component
     ```
 
     **Note:**
 
     The chart of interest is `datawire/ambassador`, which will install `Ambassador Edge Stack` on the cluster. Please visit the [ambassador-chart](https://github.com/datawire/ambassador-chart) page, for more details about this chart.
-3. Then, open and inspect the `03-setup-ingress-controller/assets/manifests/ambassador-values-v6.7.13.yaml` file provided in the `Starter Kit` repository, using an editor of your choice (preferably with `YAML` lint support). For example, you can use [VS Code](https://code.visualstudio.com):
+3. Then, open and inspect the `03-setup-ingress-controller/assets/manifests/ambassador-values-v6.9.3.yaml` file provided in the `Starter Kit` repository, using an editor of your choice (preferably with `YAML` lint support). For example, you can use [VS Code](https://code.visualstudio.com):
 
     ```shell
-    code 03-setup-ingress-controller/assets/manifests/ambassador-values-v6.7.13.yaml
+    code 03-setup-ingress-controller/assets/manifests/ambassador-values-v6.9.3.yaml
```

Comment:
I still see the old ambassador values file in the `assets/manifests` folder. I don't see the new one: `ambassador-values-v6.9.3.yaml`.